src/Stepper/Stepper.tsx

```diff
@@ -39,29 +38,27 @@ export default class Stepper extends React.PureComponent {
   static cssClass = cssClass;
 
   _renderSidebar(style) {
-    const {currentStep, steps, seekable, onStepClick, className} = this.props;
+    const {currentStep, steps, onStepClick, className} = this.props;
 
     return (
-      <div style={style}>
-        <ul className={cssClass.STEPS_DISPLAY}>
-          {steps.map((step, idx) =>
-            <li key={idx}>
-              <Step
-                className={className}
-                title={step.title}
-                description={step.description}
-                index={idx}
-                success={step.success}
-                current={idx === currentStep}
-                warning={step.warning}
-                optional={step.optional}
-                seekable={seekable}
-                onClick={onStepClick}
-              />
-            </li>
-          )}
-        </ul>
-      </div>
+      <ul style={style} className={cssClass.STEPS_DISPLAY}>
+        {steps.map(step =>
+          <li key={step.id}>
+            <Step
+              className={className}
+              title={step.title}
+              description={step.description}
+              label={step.label}
```

Comment:
this can just be the string version of the step's index + 1 (or if you want the labels to be customisable, let's at least default to `index + 1` if the client hasn't specified a label)